packages/liveblocks-devtools/src/devtools/tabs/yjs/index.tsx

```diff
@@ -228,7 +230,7 @@ function YjsAwareness({ className, ...props }: ComponentProps<"div">) {
           className={cx(className, "absolute inset-0 flex h-full flex-col")}
           {...props}
         >
-          <YjsTree data={presence} />
+          <JsonTree data={awareness} />
```

Comment:
We used to pass the `presence` array here which doesn't display Y.js awareness but the presence information. I've updated to pass the awareness array instead and use a `JsonTree` to view the data.

<img width="863" alt="Screenshot 2024-03-29 at 12 09 26 PM" src="https://github.com/liveblocks/liveblocks/assets/35807215/0c43a3a0-1363-4c57-964d-af515be0a9e4">

Reply:
Do we want to keep properties like `__yjs_clientid` visible? I think `YjsTree` was responsible for hiding them.

Reply:
I think it makes sense to at least display the client id because it might be useful with some kind of debugging? 

Values inside `__yjs` is displayed in the `Awareness` tab - so maybe we could also display this client id inside the `Awareness` tab that is purely dedicated to `Yjs`?

Reply:
Adding it to the `Awareness` tab would make the most sense to me (always keeping all __yjs direct values hidden), displayed differently so it's clear that it's not awareness. Maybe similarly to the presence IDs like `(#216)` next to `Me` in your screenshot?

Reply:
Going to work on this on a different PR so this PR isn't blocked!